src/Aspire.Hosting/Aspire.Hosting.csproj

```diff
@@ -68,6 +68,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Aspire.Hosting.Tests" />
     <InternalsVisibleTo Include="Aspire.Hosting.Containers.Tests" />
+    <InternalsVisibleTo Include="Aspire.Hosting.Milvus.Tests" />
```

Comment:
We need to remove this IVT. We can't have `Aspire.Hosting` having InternalsVisibleTo all the extension tests. We need to find a way to test extension libraries without Aspire.Hosting knowing about them.

If you need help on how to do this, can you tell me what internal APIs the current tests need?